_posts/2016-08-25-hybox-names.markdown

```diff
@@ -0,0 +1,38 @@
+
+2. **code components.** Hydra’s technical architecture has many components and layers, and an explicit Hydra-in-a-Box goal is to contribute to the overall architecture and code ecosystem. Note that a site may choose to run these HyBox components but *not *run HyBox in its turnkey form, just as many sites now say they run "sufia-based repositories." Under the theory that a rising tide raises all ships, the project team’s focus has been to locate functionality in the right places within the Hydra stack, and *not *to create a product-specific code base where we can avoid it. Accordingly, we are **not** using HyBox or Hydra-in-a-Box as the name of code components, in order to encourage re-use in other projects, broad adoption, and code contribution & maintenance from other Hydra efforts.
+
+Therefore, for new code produced by the Hydra-in-a-Box effort, we have first sought to locate it where it rightfully belongs in the stack (e.g., Sufia, an independent gem, or even Fedora). Our hope is that the work we’re doing is general and modular enough that other Hydra efforts can and will make use of our code components--e.g., it would be excellent for [the Avalon Media System](http://www.avalonmediasystem.org/) to make use of (and contribute back to!) our components when it needs similar functionality. 
+
+3. **turnkey repository product,** locally managed. Building on what the Hydra community has learned working on Sufia to make it easy to install and maintain, we expect to provide a complete, configurable, turnkey product--more akin to Avalon, DSpace, or even ContentDM. We need a name for this product analogous to those systems, and with broad market appeal and strong branding identity. We have put a lot of thought into requirements for a good name: 
+
+    * We want something clearly related to the Hydra project to leverage its brand, and to invite uptake and code contributions / maintenance. Further, we want adopters of the product to know it is part of the Hydra community effort, in order to provide an on-ramp for them to join / participate in the wider community effort. 
+
+    * We want something clearly distinct from the Hydra community & repository / application framework. (Just calling it "Hydra" wouldn’t work.)
+
+    * Hydra-in-a-Box and HyBox both seem poor choices here: the former is clunky and overly long. The latter is perhaps too insider-y and lacks the natural power and mass-market appeal that a name like "Avalon" brings.
+
+    * Also, Hydra-in-a-Box implies the full functionality and flexibility of the Hydra technical framework is present in the product, but all nicely wrapped up in one package. This is not true, as we’ll be making opinionated selections, based on the community’s requirements, about functionality to include. As such, we’re looking for a name that connotes one is getting a particular flavor or profile of Hydra.
```

Comment:
Tighten "connotes one is getting a particular flavor or profile" to "connotes a particular flavor or profile?"